web3-dev/smart-contracts/autonomous-smart-contract.rst

```diff
@@ -0,0 +1,61 @@
+.. _autonomous-smart-contract:
+
+Autonomous Smart Contract
+=========================
+
+Massa smart contracts have the unique feature of being able to autonomously define when they will be active, without external actors. By the end of this page, you will have a basic understanding of
+
+- The mechanisms behind autonomous smart contracts
+- How to use the sendMessage(...) ABI in order to make your smart contracts autonomous
+- Provided autonomous smart contracts examples
+
+Mechanism in Massa network
+--------------------------
+
+This system works by emitting messages which will schedule an execution. To do so, the following must be specified:
+
+- The address of the SC to be executed
+- The function to be executed
+- The desired execution window
+- (Optional) The activation triggers
+
+Normally, when you submit an operation, the operation is sent in the operation pool and in the future included in a block to be executed: you pay the gas at execution time.
+With autonomous SCs, the message is emitted inside of the operation and and you pay the gas for the message execution upfront. The message is then sent to another pool called the async pool.
+
+This pool is deterministic by nature, as it's filled with information that comes from operations included in blocks which is processed by every node of the network.
```

Comment:
```suggestion
This pool is deterministic in nature, as it's filled with information that comes from operations included in blocks which is processed by every node of the network.
```

both are correct, but to me, this change reads more naturally.

Alternatively: "The content of this pool is inherently deterministic, as it's..."

...but this suggestion is optional to the PR. Feel free to close without change.